source/templates/pmc-member-discuss.txt

```diff
@@ -0,0 +1,17 @@
+To: private@[project].apache.org
+Subject: [DISCUSS] New PMC member: [Candidate Name]
+
+I propose to add [Candidate Name] as a PMC member.
+
+[State reasons that you believe they are a good candidate]
+
+[State contribution statistics, if available and relevant.]
+
+PMC members are our peers, who help drive the direction of this
+project. They will be owners not just of the code, but of the project
+and its brand.
+
+Please remember that should this candidate be elected, they will have
+access to our archives, and be able to read this, so phrase your
+comments constructively with this in mind.
```

Comment:
I'm not sure if this is universally true.  At least for the Arrow project I don't believe I am able to browse the archives before I was accepted.  Though I agree things should be kept constructive in either case and so I don't think it's harmful.

Reply:
No, not *before* you're accepted, but once you are elected as a PMC member, you do immediately have that access. The point here is that people need to not be jerks, because the person they're talking about will eventually be able to see it. Of course, they shouldn't be jerks anyway ...